src/async.rs

```diff
@@ -478,8 +474,8 @@ impl<S: Sleeper> AsyncClient<S> {
     }
 }
 
-fn is_status_retryable(status: reqwest::StatusCode) -> bool {
-    RETRYABLE_ERROR_CODES.contains(&status.as_u16())
+fn is_status_retryable(status: i32) -> bool {
```

Comment:
If `async_minreq` project is open to it would be nice to have a similar `StatusCode` enum in that project to use here and also instead of your 299 value above.